.github/workflows/test.yaml

```diff
@@ -8,17 +8,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
-      - name: install dependencies
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '2.7'
+      - name: Install dependencies
         run: |
-          sudo apt-get remove -y python3-libxml2
-          sudo apt-get install -y libxml2-dev python-dev python-libxml2
-          curl https://bootstrap.pypa.io/pip/2.7/get-pip.py --output get-pip.py
-          python get-pip.py
-          pip --version
-          python -m pip install --upgrade --no-deps --force-reinstall lxml
           python -m pip install \
-            future \
-            pep8 \
             pyserial \
             tox
+      - run: rm chirp/drivers/generic_xml.py
```

Comment:
I would rather just remove it from the tree than have the test stuff silently behave differently here versus locally. I have a commit on the py3 branch to do it, I'll pull it over.